### PR TITLE
Feature/order list

### PR DESCRIPTION
**Pull request by maharabhossain1**

### CODE REVIEW

File: components/order-list.jsx [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+        <div className="min-w-max"> {order.total_quantity}</div>
+      </td>
+      <td className="p-2">
+        <div className="min-w-max"> {`${order.total_weight} (${order.unit})`} </div>
+      </td>
+      <td className="p-2">
+        <OrderStatusBadge label={statusType.badgeTitle} variant={statusType.color} />
+      </td>
+      <td className="p-2">
+        <div className="min-w-max"> {formatDate(order.expected_delivery_date, false)}</div>
```
Comment (ShehabAhmedSayem) [inline components/order-list.jsx]: instead of passing false, create a separate function called `formatDateTime`

File: components/order-list.jsx [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+      </td>
+    </tr>
+  );
+};
+
+const ItemShortDetails = ({ itemDetails }) => {
```
Comment (ShehabAhmedSayem) [inline components/order-list.jsx:49]: We can reuse this in the order details page. So creating a separate component file should be better.

Comment (ShehabAhmedSayem) [inline components/order-list.jsx:49]: We can reuse this in the order details page. So creating a separate component file should be better.

File: lib/utils.js [hunk 3]
```diff
@@ -37,16 +37,20 @@ export const uuidv4 = () => {
...
-  const minutes = date.getMinutes();
-  const ampm = hours >= 12 ? 'PM' : 'AM';
-  const formattedHours = hours % 12 || 12;
-
   const day = date.getDate();
   const month = date.toLocaleString('default', { month: 'short' });
   const year = date.getFullYear();
 
-  return `${formattedHours}:${minutes} ${ampm} | ${day} ${month}, ${year}`;
+  if (includeTime) {
```
Comment (ShehabAhmedSayem) [inline lib/utils.js]: create a separate function for `formatDateTime`.

Comment (ShehabAhmedSayem) [inline lib/utils.js]: create a separate function for `formatDateTime`  and use `formatDate` function to get the date and do the additional time related code in the `formatDateTime` function.

**Review by ShehabAhmedSayem — CHANGES_REQUESTED** [review]

File: components/order-list.jsx (hunk 2 above)
Comment (maharabhossain1) [reply components/order-list.jsx:49]: already created for the next MR vai. cause this component is used in the order details page too.

**Pull request merged**